src/Compilers/Core/Portable/FileSystem/FileUtilities.cs

```diff
@@ -366,6 +366,29 @@ internal static DateTime GetFileTimeStamp(string fullPath)
             {
                 return File.GetLastWriteTimeUtc(fullPath);
             }
+            catch (IOException)
```

Comment:
I used FileUtilities since that was the one already used in this file. just looked PerformIO which seems very similar to FileUtilites, not sure why there is 2 utilities which does same thing.